controllers/osbuildenvconfig_controller.go

```diff
@@ -90,6 +90,8 @@ const (
 	envoyProxyCertsDir = "/etc/certs"
 )
 
+var GlobalComposerComposerAPIServiceName = composerComposerAPIServiceName
```

Comment:
Why is it a `var`? Do we want to be able to change it? If not, you can just expose the const by renaming it